diff --git a/src/modules/graphics/opengl/Framebuffer.cpp b/src/modules/graphics/opengl/Framebuffer.cpp
index 93575d70b..649b4f18f 100644
--- a/src/modules/graphics/opengl/Framebuffer.cpp
+++ b/src/modules/graphics/opengl/Framebuffer.cpp
@@ -34,6 +34,7 @@ namespace opengl
 		virtual void bindFBO(GLuint) {}
 	};
 
+#if 0
 	struct FramebufferStrategyGL3 : public FramebufferStrategy {
 		virtual GLenum createFBO(GLuint& framebuffer, GLuint& depthbuffer, GLuint& img, int width, int height)
 		{
@@ -81,6 +82,7 @@ namespace opengl
 			glBindFramebuffer(GL_FRAMEBUFFER, framebuffer);
 		}
 	};
+#endif
 
 	struct FramebufferStrategyEXT : public FramebufferStrategy {
 
@@ -135,7 +137,9 @@ namespace opengl
 	
 	FramebufferStrategy strategyNone;
 	
+#if 0
 	FramebufferStrategyGL3 strategyGL3;
+#endif
 	
 	FramebufferStrategyEXT strategyEXT;
 	
@@ -144,7 +148,7 @@ namespace opengl
 	Framebuffer::Framebuffer(int width, int height) :
 		width(width), height(height)
 	{
-		strategy = NULL;
+		strategy = &strategyEXT;
 
 		float w = static_cast<float>(width);
 		float h = static_cast<float>(height);
@@ -160,7 +164,7 @@ namespace opengl
 		vertices[1].s = 0;     vertices[1].t = 0;
 		vertices[2].s = 1;     vertices[2].t = 0;
 		vertices[3].s = 1;     vertices[3].t = 1;
-
+#if 0
 		if (!strategy) {
 			if (GLEE_VERSION_3_0 || GLEE_ARB_framebuffer_object)
 				strategy = &strategyGL3;
@@ -169,6 +173,7 @@ namespace opengl
 			else
 				strategy = &strategyNone;
 		}
+#endif
 
 		loadVolatile();
 	}
diff --git a/src/modules/graphics/opengl/Framebuffer.h b/src/modules/graphics/opengl/Framebuffer.h
index c8faae295..3d8fc7f20 100644
--- a/src/modules/graphics/opengl/Framebuffer.h
+++ b/src/modules/graphics/opengl/Framebuffer.h
@@ -9,6 +9,16 @@
 #include <common/math.h>
 #include "GLee.h"
 
+#ifdef GL_EXT_framebuffer_object
+#undef GL_EXT_framebuffer_object
+#endif
+
+#ifndef GL_GLEXT_PROTOTYPES
+#define GL_GLEXT_PROTOTYPES
+#endif
+
+#include <SDL/SDL_opengl.h>
+
 namespace love
 {
 namespace graphics
diff --git a/src/modules/graphics/opengl/SpriteBatch.h b/src/modules/graphics/opengl/SpriteBatch.h
index e6290d6fc..bf0340ff5 100644
--- a/src/modules/graphics/opengl/SpriteBatch.h
+++ b/src/modules/graphics/opengl/SpriteBatch.h
@@ -33,6 +33,14 @@
 #include <graphics/Volatile.h>
 
 // OpenGL
+#ifdef GL_VERSION_1_5
+#undef GL_VERSION_1_5
+#endif
+
+#ifndef GL_GLEXT_PROTOTYPES
+#define GL_GLEXT_PROTOTYPES
+#endif
+
 #include "GLee.h"
 #include <SDL/SDL_opengl.h>
 
